documentation/MIGRATION-guide-for-next-generation-management-libraries.md

```diff
@@ -1,12 +1,12 @@
-## Guide for migrating to the next generation of Azure JavaScript SDK for Management Libraries
+# Guide for migrating to the next generation of Azure JavaScript SDK for Management-Plane Libraries
 
-This document is intended to help users migrate Javascript/Typescript SDK for management libraries to the next-generation.
+This document is intended to show the customers of the Javascript/Typescript management-plane libraries how to migrate their code to use the next-generation.
```

Comment:
i think the term management-plane is not encouraged on public docs. We use client and management libraries instead (see https://azure.github.io/azure-sdk/releases/2020-10/index.html) Management-plane is more like an internal term .

Reply:
I see. Ok, I renamed `management-plane` to `management`.